jax/_src/lib/mlir/dialects/__init__.py

```diff
@@ -17,6 +17,7 @@
 import jaxlib.mlir.dialects.chlo as chlo
 import jaxlib.mlir.dialects.mhlo as mhlo
 import jaxlib.mlir.dialects.func as func
+import jaxlib.mlir.dialects.ml_program as ml_program
```

Comment:
You'll need to make this import conditional (see below), because JAX releases its python and C++ pieces at a different cadence.

Reply:
Done.